Gemfile

```diff
@@ -21,15 +21,19 @@ gem "haml-rails", '~> 0.9'
 # We should remove this if we set up vue for the frontend
 gem 'jquery-rails'
 # Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
-gem 'turbolinks', '~> 5'
-gem 'jquery-turbolinks'
+# If we are creating a single page app with vue-router, we should remove this
+gem 'turbolinks', '< 5'
+
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.5'
 # Use Redis adapter to run Action Cable in production
 # gem 'redis', '~> 3.0'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 
+gem 'jquery-datatables-rails', '~> 3.4.0'
```

Comment:
It looks like you messed up the merge a little; notice that `jquery-datatables-rails` is here twice. I moved some of the gems around when reproducing your changes in 7cd19f225df0a39951e083d54fc287fa1268ce4e so that they would be in logical (to me at least) places. E.g. `jquery-turbolinks` is below `turbolinks` (see above).